libs/api/domains/health-directorate/src/lib/health-directorate.resolver.ts

```diff
@@ -11,41 +18,39 @@ import {
 import type { User } from '@island.is/auth-nest-tools'
 import { ApiScope } from '@island.is/auth/scopes'
 import { Audit } from '@island.is/nest/audit'
-import {
-  DonationException,
-  DonorStatus,
-  DonorStatusInput,
-} from './models/organ-donation.model'
+import { DonorInput, Organ, OrganDonation } from './models/organ-donation.model'
 import type { Locale } from '@island.is/shared/types'
 import { Vaccinations } from './models/vaccinations.model'
 import { HealthDirectorateService } from './health-directorate.service'
 
 @UseGuards(IdsUserGuard, ScopesGuard)
 @Scopes(ApiScope.internal)
 @Audit({ namespace: '@island.is/api/health-directorate' })
-@Resolver()
+@Resolver(() => OrganDonation)
 export class HealthDirectorateResolver {
   constructor(private api: HealthDirectorateService) {}
 
   /* Organ Donation */
-  @Query(() => DonorStatus, {
-    name: 'HealthDirectorateOrganDonationGetDonorStatus',
+  @Query(() => OrganDonation, {
+    name: 'HealthDirectorateOrganDonation',
```

Comment:
Should be lowercase
```suggestion
    name: 'healthDirectorateOrganDonation',
```

Reply:
🙈 
![image](https://github.com/user-attachments/assets/1fcafa42-4787-47e2-9633-4a223ae74039)